cves/CVE-2009-2693.yml

```diff
@@ -43,15 +43,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Vulnerability in directory pathing system. Allowed remote users to create and overwrite files without proper permission when using ".." in directory path. 
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2010-01-28
+  announced: 
   url:
```

Comment:
Where did you find the bounty?

Reply:
From my research there was no indication of a bounty, that is why I removed the entry already present.